frontends/mit-open/src/page-components/SearchDisplay/ProfessionalToggle.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react"
+import {
+  Collapse,
+  ToggleButton,
+  ToggleButtonGroup,
+  styled,
+} from "ol-components"
+import { RiBookOpenLine, RiBriefcase3Line } from "@remixicon/react"
+
+const StyledToggleButtonGroup = styled(ToggleButtonGroup)`
+  width: 100%;
+`
+const StyledButtonGroupContainer = styled.div`
+  margin-top: 10px;
+  margin-bottom: 10px;
+`
+const ExplanationContainer = styled.div`
+  margin: 10px;
+  font-size: 0.875em;
+  min-height: 35px;
+`
+
+const StyledRiBookOpenLine = styled(RiBookOpenLine)`
+  height: 20px;
+  margin-right: 3px;
+  margin-left: -2px;
+`
+
+const StyledRiBriefcase3Line = styled(RiBriefcase3Line)`
+  height: 20px;
+  margin-right: 3px;
+  margin-left: -2px;
+`
+
+const StyledToggleButton = styled(ToggleButton)(({ theme }) => ({
+  width: "100%",
+  height: "50px",
+  backgroundColor: theme.custom.colors.lightGray2,
+  color: theme.custom.colors.darkGray2,
+  "&.Mui-selected": {
+    backgroundColor: theme.custom.colors.white,
+    "&:hover": {
+      backgroundColor: theme.custom.colors.white,
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.custom.colors.white,
+  },
+}))
+
+const ViewAllButton = styled(StyledToggleButton)`
+  border-right: 2px solid;
+  border-color: ${({ theme }) => theme.custom.colors.silverGrayLight};
+  width: 25%;
+`
+const AcademicButton = styled(StyledToggleButton)`
+  border-right: 2px solid;
+  border-color: ${({ theme }) => theme.custom.colors.silverGrayLight};
+  width: 35%;
+`
+const ProfesionalButton = styled(StyledToggleButton)`
+  width: 40%;
+`
+
+const ProfessionalToggle: React.FC<{
+  professionalSetting: boolean | null | undefined | string
+  setParamValue: (_name: string, _rawValue: string | string[]) => void
+}> = ({ professionalSetting, setParamValue }) => {
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newValue: string,
+  ) => {
+    setParamValue("professional", newValue)
+  }
+
+  if (professionalSetting === true || professionalSetting === false) {
+    professionalSetting = professionalSetting.toString()
+  } else {
+    professionalSetting = ""
+  }
+
+  return (
+    <StyledButtonGroupContainer>
+      <StyledToggleButtonGroup
+        value={professionalSetting}
+        exclusive
+        onChange={handleChange}
+        aria-label="text alignment"
```

Comment:
seems like a mis-labeled aria

Reply:
Yes - the description is intended to have an animation. Fixing the aria